### DISCUSSION TITLE
User guide: Parameters

### DISCUSSION THREAD

**Discussion by AVHopp** [emdgroup/baybe#48] [Documentation]
The user guide for the parameters might need an overhaul but needs to checked more thoroughly.

**Comment by Scienfitz** [comment]
@AVHopp since youve opened this, is there anything concrete that needs to change? otherwise Id clsoe this to only have actually in progress / todo items here

**Reply by Scienfitz** [reply]
@AVHopp *ping*

**Reply by AVHopp** [reply]
I think this is a legacy item. The only thing that was still open was the `TaskParameter`, but as we decided that this will be discussed in the TL userguide, I'll close this.
